src/BGhostview.cpp

```diff
@@ -45,15 +45,13 @@ BGhostview::BGhostview( BRect frame, const char *name )
 		strcpy(psTmp,"/boot/apps/Ghostscript/");
 		}
 	else {
```

Comment:
this if/else can be completely removed, I think? we never want to use /boot/apps/Ghostscript as a temp directory.

Reply:
Should be good now, let me know if it works for you or if it crashes when changing pages as well.
If it works I think I will tag a new release?